pandas/core/groupby/ops.py

```diff
@@ -97,6 +98,7 @@ def __init__(
         self.group_keys = group_keys
         self.mutated = mutated
         self.indexer = indexer
+        self.dropna = dropna
```

Comment:
@arw2019 this attribute isnt used anywhere except for in the test added in this PR.  is it still needed?  is it part of a precursor to something on the horizon?